frontend/src/libs/components/author/author.tsx

```diff
@@ -0,0 +1,48 @@
+import { type FC } from 'react';
+
+import { Avatar } from '../avatar/avatar.jsx';
+import styles from './styles.module.scss';
+
+type Properties = {
+  name?: string;
+  followers?: number;
+  rating?: number;
+  date?: string;
+  readingTime?: string;
+  genre?: string;
+};
+
+const Author: FC<Properties> = ({
+  name = 'Todd Demoer',
+  followers = 10,
+  rating = 700,
+  date = 'May 28',
+  readingTime = '7 min',
+  genre = 'Fiction',
+}) => {
+  return (
+    <>
+      <div className={styles.container}>
+        <div className={styles.avatar_wrpr}>
```

Comment:
I think it would be better to name the class as avatarWrapper

Reply:
This is author-component from task/wr-53. I have merge those branch when worked on my task. I will merge those branch again with changes after review. I already linked your comments to wr-53.

Reply:
It had already been renamed for avatarWrpr, so as other supposed changes were made too. @b0bbobovich, I guess to should merge the latest branch changes

Reply:
In any case, I will check all supposed changes in my branch once more

Reply:
We've missed it, but rename it to avatarWrapper please

Reply:
@kovAnya I will rename all  needed classes to "...Wrapper" instead "...Wrpr"

Reply:
Oo. Sorry this is old folder from first merge with wr-53. I thougth that it will be deleted after merging with updated wr-53 branch (author-component). But it seems like it is stayed in my branch somehow.  I already changed naming in the pages/article/components/author.tsx and deleted this useless common author-component.